tooling/lsp/src/requests/completion/auto_import.rs

```diff
@@ -127,69 +142,73 @@ fn module_def_id_to_reference_id(module_def_id: ModuleDefId) -> ReferenceId {
     }
 }
 
-/// Computes the path of `module_id` relative to `current_module_id`.
-/// If it's not relative, the full path is returned.
+/// Returns the path to reach an item inside `module_id` from inside `current_module_id`.
+/// Returns a relative path if possible.
 fn module_id_path(
-    module_id: &ModuleId,
+    module_id: ModuleId,
```

Comment:
```suggestion
    target_module_id: ModuleId,
```

Can we rename this to make it more distinct from `current_module_id`?

Reply:
Makes sense, done!